### DISCUSSION TITLE
RustDesk Client very fuzzy about network setting string - other way to distribute them?

### DISCUSSION THREAD

**Discussion by reox** [rustdesk/rustdesk#10108] [Q&A]
I had the following issue while trying to setup remote desktop on my self-hosted server with a family member: I send a mail with the network settings, i.e., the reversed base64 string of the settings JSON. They copied it from the mail and tried to paste but always got "Invalid server configuration". Eventually we found out there was a whitespace copied after the string, which caused this error. Eventually we found out, that selecting the string in the mail client also selected the whitespace.

I wondered it there is any other way to distribute the network settings to other people in the self-hosted setting, without having to take care that everything is strictly copied, i.e., extra whitespace etc?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/763, go to [discord](https://discord.gg/nDceKgxnkV) please, search there first, some users might have asked similar questions. Usually nobody will help you with build or self-hosting here.

**Comment by rustdesk** [comment]
> I wondered it there is any other way to distribute the network settings to other people in the self-hosted setting, without having to take care that everything is strictly copied, i.e., extra whitespace etc?

Custom client in the self-hosting Pro. https://github.com/rustdesk/rustdesk/discussions/8981
